angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead-selector/sprk-masthead-selector.module.ts

```diff
@@ -0,0 +1,18 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { SprkIconModule } from '../../sprk-icon/sprk-icon.module';
+import { SprkLinkDirectiveModule } from '../../../directives/sprk-link/sprk-link.module';
+import { SprkMastheadSelectorComponent } from './sprk-masthead-selector.component';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    SprkIconModule,
```

Comment:
`sprkText` directive needs to be imported here to work in the masthead selector. See this comment: https://github.com/sparkdesignsystem/spark-design-system/pull/3801/files#r574872430